packages/app/schema/nl/corona_app.json

```
@@ -0,0 +1,45 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "title": "nl_corona_app",
```

Other schema's use `national` as prefix, not `nl`:

```suggestion
  "title": "national_corona_app",
```

---

@VWSCoronaDashboard4 The plan was to start using nl/vr/gm as the new prefix, because the file names have changed from national to nl. Only I didn't find time yet to change all of the titles. The newer schema's use the nl_ prefix